fuzz/read-metadata.cpp

```diff
@@ -0,0 +1,65 @@
+// ***************************************************************** -*- C++ -*-
+// exifprint.cpp
+// Sample program to print the Exif metadata of an image
+
+#include <exiv2/exiv2.hpp>
+
+#include <iostream>
+#include <iomanip>
+#include <cassert>
+
+// https://github.com/Exiv2/exiv2/issues/468
+#if defined(EXV_UNICODE_PATH) && defined(__MINGW__)
```

Comment:
Please drop all of this unicode path handling, this is only relevant for MinGW where ASAN doesn't work.